orchestrator/careplanservice/service.go

```diff
@@ -443,3 +445,51 @@ func (s Service) baseUrl() *url.URL {
 func getResourceType(resourcePath string) string {
 	return strings.Split(resourcePath, "/")[0]
 }
+
+func (s *Service) ensureSearchParameterExists() {
+	searchParamID := "CarePlan-subject-identifier"
+
+	// Define the SearchParameter
+	searchParam := fhir.SearchParameter{
+		Id:          &searchParamID,
+		Url:         "http://zorgbijjou.nl/SearchParameter/CarePlan-subject-identifier",
```

Comment:
out of scope for this change, but we should probably change this to Shared Care Planning.